### DISCUSSION TITLE
Need patch for Multiselect Products in Quotes while adding

### DISCUSSION THREAD

**Discussion by JEEVAN28** [YetiForceCompany/YetiForceCRM#14845] [How to] [answered]
Hi
I am looking for this patch available on gitdeveloper in Quote / Invoice module

![image](https://user-images.githubusercontent.com/12723595/122411843-b861c980-cfa2-11eb-9532-4c6a16289fa4.png)

Please advise.

Regards
Jeevan Pendhari

**Comment by vovpff** [comment, marked as answer]
Wait for next release and this feature will be available
